src/foam/comics/DAOControllerView.js

```diff
@@ -220,6 +224,18 @@ foam.CLASS({
       } else {
         this.onEdit(null, null, obj.id);
       }
+    },
+
+    function fetchDaoCount() {
+      this.data.data.select(this.COUNT()).then((count) => {
+        if ( count.value === 0 ) {
```

Comment:
This would be clearer:

if ( count.value === 0 && this.data.emptyStateView ) {
  this.start().tag(this.data.emptyStateView).end();
} else {
  this.start().add(this.data.border).end();
}